.github/workflows/ci.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         run: flake8 . || true # preventing CI failing while flake8 issues exist
 
       - name: flake8 validating correct file(s)
-        run: flake8 asv/main.py asv/repo.py asv/results.py asv/runner.py asv/statistics.py
\ No newline at end of file
+        run: flake8 test/conftest.py test/test_dev.py test/test_feed.py test/test_repo_template/asv_test_repo/__init__.py
```

Comment:
What is the goal of this change?

Reply:
I wanted flake8 to check the files I changed but I will remove this as it's not necessary